WooCommerce/Classes/ViewRelated/Orders/Order Details/Issue Refunds/IssueRefundViewController.swift

```diff
@@ -0,0 +1,141 @@
+import UIKit
+
+/// Screen that allows the user to refund items (products and shipping) of an order
+///
+final class IssueRefundViewController: UIViewController {
```

Comment:
When the project is close to completion, we should probably think about renaming them so the names are closely related. 😅